### DISCUSSION TITLE
SVGWithLogoAndCustomShapesOptions Logo problem

### DISCUSSION THREAD

**Discussion by oskay12** [chillerlan/php-qrcode#285] [Q&A] [answered]
Hey, as i mentioned SVGWithLogoAndCustomShapesOptions discussion i can not implement logo to qr and also the dots are not heart shaped. Also i checked my version its 5.0.2. Actually what i want is an changeable dots and png or svg logo qr im saving base64 code to database for showing them at the page.

**Comment by codemasher** [comment, marked as answer]
So have you tried the example from the 5.x branch then? This should work as expected https://github.com/chillerlan/php-qrcode/blob/v5.0.x/examples/svgWithLogoAndCustomShapes.php

**Reply by oskay12** [reply]
Thank you very much im appreciate
